api/krusty/localizer/runner_test.go

```diff
@@ -452,13 +452,11 @@ func TestResourcesRepoNotFile(t *testing.T) {
 
 	err := localizer.Run(fsActual, testDir.String(), "", testDir.Join("dst"))
 
-	const readmeErr = `mapping values are not allowed in this context`
-	fileErr := fmt.Sprintf(`invalid resource at file "%s": MalformedYAMLError:`, repo)
+	fileErr := fmt.Sprintf(`invalid resource at file "%s": MalformedYAMLError`, repo)
 	rootErr := fmt.Sprintf(`unable to localize root "%s": unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization'`, repo)
 	var actualErr PathLocalizeError
 	require.ErrorAs(t, err, &actualErr)
 	require.Equal(t, repo, actualErr.Path)
-	require.ErrorContains(t, actualErr.FileError, readmeErr)
 	require.ErrorContains(t, actualErr.FileError, fileErr)
 	require.ErrorContains(t, actualErr.RootError, rootErr)
```

Comment:
I removed the `readmeErr` that's specific to the contents of the kustomize repo README.md and irrelevant to our test, which just expects that the README is invalid yaml.

The over-specificity of this test previously failed the ci pipeline on master: https://github.com/kubernetes-sigs/kustomize/actions/runs/4018832421/jobs/6904935339